SSA/Projects/InstCombine/LLVM/Semantics.lean

```diff
@@ -11,38 +11,38 @@ namespace LLVM
 The ‘and’ instruction returns the bitwise logical and of its two operands.
 -/
 def and? {w : Nat} (x y : BitVec w) : Option <| BitVec w :=
- .some <| x &&& y
+ some <| x &&& y
```

Comment:
I'm curious about these changes, what's the reason for dropping the `.`s? is this aesthetical or functional?

Reply:
I am not sure. Maybe @alexkeizer knows why this change was implemented?

Reply:
To me it seems to just be a readability improvement.

Reply:
Yes, because Option is globally open, we choose to drop the `.`

Reply:
So I can merge this?

Reply:
It was indeed purely aesthetical